pkg/validations/commandvalidations/kubeconfig.go

```diff
@@ -0,0 +1,20 @@
+package commandvalidations
+
+import (
+	"fmt"
+
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/validations"
+)
+
+func ValidateKubeconfigPath(clusterName string) error {
```

Comment:
same comment here about having this in `pkg/kubeconfig`

Reply:
Moved to pkg/kubeconfig